datadog/dogstatsd/base.py

```diff
@@ -147,7 +145,7 @@ def __init__(
         host=DEFAULT_HOST,                      # type: Text
         port=DEFAULT_PORT,                      # type: int
         max_buffer_size=None,                   # type: None
-        flush_interval=DEFAULT_BUFFERING_FLUSH_INTERVAL,  # type: float
+        flush_interval=DEFAULT_FLUSH_INTERVAL,  # type: float
         disable_aggregating=True,               # type: bool
```

Comment:
I missed this in the last PR, but should we call this `disable_aggregation`?
aggregation seems to be the term we use throughout.

Reply:
We have a function called [disable_aggregation](https://github.com/DataDog/datadogpy/blob/enable-buffering-aggregation-simultaneously/datadog/dogstatsd/base.py#L632) so the variable name is a bit different

Reply:
we seem to have the same conflict with `disable_buffering` so I think it should be fine. Though I will admit it's a bit confusing having multiple re-definitions of the same name. (I think this confused me in a previous PR comment)

Reply:
Sounds good, I can make the change